framework/include/interfaces/DependencyResolverInterface.h

```diff
@@ -122,7 +122,7 @@ DependencyResolverInterface::cyclicDependencyError(CyclicDependencyException<T2>
   const auto cycle = e.getCyclicDependencies();
   std::vector<std::string> names(cycle.size());
   for (const auto i : index_range(cycle))
-    names[i] = static_cast<T>(cycle[i])->name();
+    names[i] = static_cast<T>(cycle[i])->typeAndName();
```

Comment:
I kind of liked how concise the output was before. Seems like this could be a lot of text for a large cycle. I also don't know why there is a trailing slash after the name in `typeAndName()`

Reply:
There isn't a trailing slash? It's `ObjectType "name"`:

```
  /**
   * Get the class's combined type and name; useful in error handling.
   * @return The type and name of this class in the form '<type()> "<name()>"'.
   */
  std::string typeAndName() const
  {
    return type() + std::string(" \"") + name() + std::string("\"");
  }
```

It made it a lot easier for me to debug, but I'm not invested. Will remove

Reply:
Lol, read the string wrong. If this was really helpful for you, then let's keep it